class.jetpack.php

```diff
@@ -728,6 +725,23 @@ function( $callable ) {
 		add_action( 'script_loader_tag', array( $this, 'script_add_async' ), 10, 3 );
 	}
 
+	/**
+	 * Runs after all the plugins have loaded but before init.
+	 *
+	 */
+	function after_plugins_loaded() {
+		$terms_of_service = new \Automattic\Jetpack\Terms_Of_Service();
```

Comment:
I suggest that we always keep the leading backslash off. It's unnecessary, and when omitted, allows for us to magically mock classes when used in other namespaces.